tests/test_conf.py

```diff
@@ -9,13 +10,20 @@
 def test_parse_value():
     from pgtoolkit.conf import parse_value
 
+    def random_upper(v: str) -> str:
+        return "".join(choice((str.upper, str.lower))(char) for char in v)
+
     # Booleans
-    assert parse_value("on") is True
-    assert parse_value("off") is False
-    assert parse_value("true") is True
-    assert parse_value("false") is False
-    assert parse_value("yes") is True
-    assert parse_value("'no'") is False
+    for v, r in [
+        ("on", True),
+        ("off", False),
+        ("true", True),
+        ("false", False),
+        ("yes", True),
+        ("'no'", False),
+    ]:
+        assert parse_value(v) is r
+        assert parse_value(random_upper(v)) is r
```

Comment:
Not sure we need this extra complexity:
- in my experience loops in tests are hard to debug (might be quite easy in this case, but in general it's not); and that's why parametrized tests exist (by the way, it'd be nice to refactor this test to make it parametrized, but that's another issue)
- the fact that the randomized value is not readily available (one would need to change this code through an extra variable to see it) is also problematic
- so in short, I think we could keep the previous logic and just change e.g. `true` into `TRUE`, `no` into `No`; this would be largely enough for coverage